cfgov/jinja2/v1/_includes/molecules/notification.html

```diff
@@ -47,3 +47,27 @@
   </div>
 
 {% endmacro %}
+
+{% if value %}
+
+    {#
+      TODO: This if/else is for removing the explanation if it's empty HTML.
+            Investigate a better way to do this that doesn't repeat the call.
+    #}
+    {% if ( value.explanation | trim ) == '<div class="rich-text"><p><br/></p></div>' %}
```

Comment:
@chosak Quite certain there's a better way than this but I don't know what it is. Do you? I'm checking that the explanation is an empty rich text block here and excluding it from the render call if needed. I tried `{% set value.explanation = none %}` and got an error (`cannot assign attribute on non-namespace object`).


cannot assign attribute on non-namespace object
--


.

Reply:
@anselmbradford this is a known wontfix issue ([1](https://github.com/wagtail/wagtail/issues/3111), [2](https://github.com/wagtail/wagtail/issues/4046)) for Wagtail, addressed in the new Wagtail 2.x rich text editor, which unfortunately we can't move to right now.

We could maybe do something like [this](https://github.com/wagtail/wagtail/issues/2336#issuecomment-396536853). Is there anywhere else on the site where we've encountered this, or need to do boolean tests on rich text fields?

Reply:
That note at the bottom…

> The html module is available only in Python 3.

Does that mean we can't use this approach yet?

Reply:
If the resolution here is to leave a `TODO` I'm fine with that, if you are.

Reply:
@anselmbradford we don't have to do it exactly as proposed in that link. The problem with the existing version is that `<div class="rich-text"><p><br/></p></div>` isn't guaranteed -- it might have zero `<br/>`s, one, multiple.

I'll take another look a little later today to see if I can find a simple workaround.

One question, though: does this have to be rich text, or could it just be text? What is the use case for rich text -- having headings? Can you be more specific about exactly which rich text elements are to be allowed in this notification?

Reply:
> One question, though: does this have to be rich text, or could it just be text? What is the use case for rich text -- having headings? Can you be more specific about exactly which rich text elements are to be allowed in this notification?

Good question. We have instances that have links. That's the only requirement that I thought necessitated rich text.